src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1425,6 +1353,100 @@ private static T GetService<T>(this IServiceProvider services, T defaultService)
             }
         };
 
+        private static IReadOnlyDictionary<TexlFunction, AsyncFunctionPtr> TabularFunctionImplementations { get; } = new Dictionary<TexlFunction, AsyncFunctionPtr>
+        {
+            {
+                BuiltinFunctionsCore.AbsT,
+                StandardErrorHandlingAsync<TableValue>(
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<TableValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
+                    targetFunction: StandardSingleColumnTable<NumberValue>(SimpleFunctionImplementations[BuiltinFunctionsCore.Abs]))
+            },
+            {
+                BuiltinFunctionsCore.Boolean_T,
+                StandardErrorHandlingAsync<TableValue>(
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<TableValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
+                    targetFunction: StandardSingleColumnTable<StringValue>(SimpleFunctionImplementations[BuiltinFunctionsCore.Boolean]))
+            },
+            {
+                BuiltinFunctionsCore.BooleanN_T,
+                StandardErrorHandlingAsync<TableValue>(
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<TableValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
+                    targetFunction: StandardSingleColumnTable<NumberValue>(SimpleFunctionImplementations[BuiltinFunctionsCore.BooleanN]))
+            },
+            {
+                BuiltinFunctionsCore.CharT,
+                StandardErrorHandlingAsync(
```

Comment:
>StandardErrorHandlingAsync [](http://example.com/codeflow?start=16&length=26)

Most of these are                 StandardErrorHandlingAsync<TableValue>, 

should this be too?

Reply:
Created another wrapper for tabular functions - code is much simpler now :-)